Classes/Service/ContentEditableWrapperService.php

```diff
@@ -155,14 +155,15 @@ public function wrapContentWithDropzone(
         $class = 't3-frontend-editing__dropzone';
 
         $dropZone = sprintf(
-            '<div class="%s" ondrop="%s" ondragover="%s" ondragleave="%s" data-new-url="%s" data-moveafter="%d" data-colpos="%d"></div>',
+            '<div class="%s" ondrop="%s" ondragover="%s" ondragleave="%s" data-new-url="%s" data-moveafter="%d" data-colpos="%d" data-defvals=\'%s\'></div>',
```

Comment:
I try to use JSON_HEX_QUOT
http://php.net/manual/en/json.constants.php#constant.json-hex-quot